src/Adapter/Module/Tab/ModuleTabUnregister.php

```diff
@@ -24,15 +24,15 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+namespace PrestaShop\PrestaShop\Adapter\Module\Tab;
```

Comment:
I put these classes in `Adapter` because I'm still using the Legacy `Tab` class 😞